tiatoolbox/__init__.py

```diff
@@ -14,20 +14,31 @@
 # along with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
-# The Original Code is Copyright (C) 2020, TIALab, University of Warwick
+# The Original Code is Copyright (C) 2021, TIALab, University of Warwick
 # All rights reserved.
 # ***** END GPL LICENSE BLOCK *****
 
 """Top-level package for TIA Toolbox."""
-from tiatoolbox import tiatoolbox
-from tiatoolbox import wsicore
-from tiatoolbox import utils
-from tiatoolbox import tools
 
+import os
 
 __author__ = """TIA Lab"""
 __email__ = "tialab@dcs.warwick.ac.uk"
 __version__ = "0.6.0"
 
+# This will set the tiatoolbox external data
+# default to be the user home folder, should work on both Window and Unix/Linux
+# C:\Users\USER\.tiatoolbox
+# /home/USER/.tiatoolbox
+
+# runtime context parameters
+rcParam = {"TIATOOLBOX_HOME": os.path.join(os.path.expanduser("~"), ".tiatoolbox")}
+
+from tiatoolbox import tiatoolbox
```

Comment:
Please we combine it in single line `from tiatoolbox import tiatoolbox, wsicore, utils, tools, models`?

Reply:
Done